src/function/function_collection.cpp

```diff
@@ -42,6 +42,8 @@ namespace function {
     {_PARAM::getFunctionSet, _PARAM::name, CatalogEntryType::SCALAR_FUNCTION_ENTRY}
 #define SCALAR_FUNCTION_ALIAS(_PARAM)                                                              \
     {_PARAM::getFunctionSet, _PARAM::alias, CatalogEntryType::SCALAR_FUNCTION_ENTRY}
+#define SCALAR_FUNCTION_ALIAS_1(_PARAM)                                                            \
+    {_PARAM::getFunctionSet, _PARAM::alias1, CatalogEntryType::SCALAR_FUNCTION_ENTRY}
```

Comment:
Are u sure about alias1 design? I don't know how to register multiple aliases. But alias1 certainly does not sound like a good design.